extensions/s3/src/main/java/io/deephaven/extensions/s3/S3Instructions.java

```diff
@@ -37,6 +36,7 @@ public abstract class S3Instructions implements LogOutputAppendable {
     private static final Duration DEFAULT_READ_TIMEOUT = Duration.ofSeconds(2);
     private static final int DEFAULT_NUM_CONCURRENT_WRITE_PARTS = 64;
     private static final int MIN_CONCURRENT_WRITE_PARTS = 1;
+    private static final Duration MIN_TIMEOUT = Duration.ofMillis(1);
```

Comment:
```suggestion
    private static final Duration MIN_READ_WRITE_TIMEOUT = Duration.ofMillis(1);
```
Let's call this `MIN_READ_WRITE_TIMEOUT` since we are not using it for other timeouts, like connection timeout, just reads and writes.